backend/src/data/repositories/city-repository.ts

```diff
@@ -0,0 +1,15 @@
+import { CityModel } from '../models';
+import { ICity } from '~/common/interfaces';
+
+class CityRepository {
+
+  public getAll(): Promise<ICity[]> {
+    return CityModel.findAll();
+  }
+
+  public createCity(cityName: Partial<ICity>): Promise<ICity> {
+    return CityModel.create(cityName);
+  }
+}
+
+export { CityRepository };
```

Comment:
```suggestion
class City {

  public getAll(): Promise<ICity[]> {
    return CityModel.findAll();
  }

  public createCity(cityName: Partial<ICity>): Promise<ICity> {
    return CityModel.create(cityName);
  }
}

export { City };
```
Rename file to `backend/src/data/repositories/city.repository.ts`
Rename all repositories this way